src/app/components/PublishersTable.jsx

```diff
@@ -29,30 +37,33 @@ export default function PublishersTable({ publishers }) {
                     )}
                   </div>
                   <div className="text-base text-gray-500">
-                    {publisher.description}
+                    {publisher.host} · {publisher.paymentsCount} payments
                   </div>
                 </div>
               </div>
             </td>
             <td className="px-6 py-6 whitespace-nowrap text-right">
-              <div className="ml-40">
-                <p className="text-lg text-gray-700 mb-0">
-                  {publisher.sats} sats
-                </p>
-                <div className="relative mt-2 w-44 ml-auto">
-                  <div className="flex items-center">
-                    <div
-                      style={{ width: `${publisher.percentage}%` }}
-                      className="ml-auto h-2 shadow-none rounded flex flex-col text-center whitespace-nowrap text-white justify-center bg-blue-500"
-                    />
-                    <span
-                      className={`ml-2 text-sm font-semibold text-blue-500`}
-                    >
-                      {publisher.percentage}%
-                    </span>
+              <span onClick={deletePublisher(publisher.id)}>delete</span>
```

Comment:
Should be () => deletePublisher(publisher.id). Now the function is invoked directly.

Reply:
👍 yep. thanks